frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -134,6 +138,68 @@ const WARNING_TYPE_RENDERER = {
             </>
         )
     },
+    replay_timestamp_invalid: function Render(warning: IngestionWarning): JSX.Element {
+        const details: {
+            timestamp: string
+            session_id: string
+        } = {
+            timestamp: warning.details.timestamp,
+            session_id: warning.details.replayRecord.session_id,
+        }
+        return (
+            <>
+                Session replay data dropped due to invalid timestamp::
+                <ul>
+                    <li>invalid timestamp: {details.timestamp}</li>
+                    <li>session_id: {details.session_id}</li>
+                </ul>
+                <div className="max-w-30 mt-2">
+                    <LemonButton
+                        type="primary"
+                        size="xsmall"
+                        to={urls.replaySingle(details.session_id)}
+                        sideIcon={<IconPlayCircle />}
+                        data-attr="skewed-timestamp-view-recording"
+                    >
+                        View recording
+                    </LemonButton>
+                </div>
+            </>
+        )
+    },
+    replay_timestamp_too_far: function Render(warning: IngestionWarning): JSX.Element {
+        const details: {
+            timestamp: string
+            session_id: string
+            daysFromNow: string
+        } = {
+            timestamp: warning.details.timestamp,
+            session_id: warning.details.replayRecord.session_id,
+            daysFromNow: warning.details.daysFromNow,
```

Comment:
Worth adding the current timestamp as well, so it's easier to think when the event was processed and the diff was computed, thinking about if I read this warning a few days after the fact, might be helpful/easier.